### PR TITLE
[GPII-2805]: Update CouchDB restoration documentation

### PR DESCRIPTION
**Pull request by natarajaya**
[Rendered README](https://github.com/gpii-ops/gpii-infra/blob/e5273bdddd77961a66ff2e72cfa6cd4a9959ab73/gcp/README.md#working-with-couchdb-data).

### CODE REVIEW

**Comment by stepanstipl** [comment]
Thanks @natarajaya, I think it looks good. Maybe it would make sense to verify that the data has been recovered successfully as part of the process? 

And although we said not to fully automated this at first step, it seems pretty lengthy & complex. I was thinking what about writing down the process in the form of commented shell script? Seems like almost all the bits are there already and a script would make it much easier to execute regularly and verify that all the steps are actually correct, and would be the first step towards fully automating this later on. 

Also if we ever really need to perform the recovery on a production environment, it would make the whole process much faster and less stressful when under pressure.

File: gcp/README.md [hunk 1]
```diff
@@ -484,11 +468,34 @@ An alternative workaround is to do a snapshot-restore cycle to move the Persiste
...
+
+Ops team must perform backup restoration test using this scenario on `gpii-gcp-stg` cluster monthly, to make sure that:
+* Automated backups are being created as expected.
+* Existing backups can be used to restore functional and consistent DB.
+* Restoration guide (this scenario) is accurate.
+
+Here are the steps:
 
-1. Choose a snapshot set that you want to restore, make sure that snapshots are present for all disks that are currently in use by CouchDB cluster.
+1. Choose a snapshot set that you want to restore (in case of a test, pick randomly from any snapshot set created in last month), make sure that snapshots are present for all disks that are currently in use by CouchDB cluster.
```
Comment (mrtyler) [inline gcp/README.md]: I like the spirit of this, but I think we should use the latest snapshot set available. If we break snapshotting the day before the restore test, I want to find out with probability ~30/30 not probability ~1/30 ;).

**Comment by mrtyler** [comment]
> Maybe it would make sense to verify that the data has been recovered successfully as part of the process?

Agreed. I think smoke tests are probably sufficient for now. In the future, it might be neat to verify that the most recent update is less than N hours old.

> what about writing down the process in the form of commented shell script?

More automation is always good, but I think this task is a little trickier than it appears. If we plan to change how this process works in the short- to medium-term, I'm not sure how much effort I would invest now.

File: gcp/README.md (hunk 1 above)
Comment (natarajaya) [reply gcp/README.md]: Good point, thanks!

**Comment by natarajaya** [comment]
@mrtyler @stepanstipl Thanks for your comments! I'll merge this as-is and work on automation part later.

**Pull request merged**